dotnet/test/AutoGen.Tests/BasicSampleTest.cs

```diff
@@ -49,12 +49,6 @@ public async Task MistralClientAgent_TokenCount()
             await Example14_MistralClientAgent_TokenCount.RunAsync();
         }
```

Comment:
Was this deletion intentional ?

Reply:
Yes, this example is too fragile to succeed with gpt-3.5 so I delete it from test